src/main/java/lotto/view/InputBonusNumberView.java

```diff
@@ -0,0 +1,30 @@
+package lotto.view;
+
+import java.util.List;
+import java.util.regex.Pattern;
+import lotto.exception.SingleNumberFormatException;
+
+public class InputBonusNumberView extends InputView {
+    private static final Pattern PATTERN = Pattern.compile("\\d+");
+    private static final String INPUT_BONUS_NUMBER_MESSAGE = "보너스 번호를 입력해 주세요.";
+
+    public Integer getValue() {
+        System.out.println(INPUT_BONUS_NUMBER_MESSAGE);
+        String result = inputValue();
+        System.out.println();
+        validateFormat(result);
+        return Integer.parseInt(result);
+    }
+
+    private void validateFormat(String inputValue) {
+        if (!PATTERN.matcher(inputValue).matches()) {
```

Comment:
```suggestion
        if (!inputValue.matches(PATTERN)) {
```
PATTERN을 String으로 선언하고 위처럼 사용하면 더 간결하게 처리할 수 있습니다!